pkg/arvo/sys/vane/eyre.hoon

```diff
@@ -2662,7 +2668,7 @@
       %~2020.10.18
     %_  $
       date.old          %~2022.8.22
-      server-state.old  [-.server-state.old *(axal (list cache-entry)) *(map @t path) +.server-state.old]
+      server-state.old  [-.server-state.old *(axal (list (unit cache-entry))) *(map @t path) +.server-state.old]
```

Comment:
I suppose this could be a `lest`, since there's no longer a way to end up with a `~` in the trie. Then we wouldn't have to use `fall` in a few places. Worth it?

Reply:
Not worth it imo